deploy/clowdapp.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ingress
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ingress
+  spec:
+    envName: ${ENV_NAME}
+    pods:
+    - name: ingress
+      image: ${IMAGE}:${IMAGE_TAG}
+      minReplicas: ${{REPLICAS}}
+      livenessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /api/ingress/v1/version
+          port: 8000
+          scheme: HTTP
+        initialDelaySeconds: 5
+        periodSeconds: 5
+        successThreshold: 1
+        timeoutSeconds: 120
+      readinessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /api/ingress/v1/version
+          port: 8000
+          scheme: HTTP
+        initialDelaySeconds: 5
+        periodSeconds: 5
+        successThreshold: 1
+        timeoutSeconds: 120
+    env:
+    - name: INGRESS_STAGEBUCKET
+      value: ${INGRESS_STAGEBUCKET}
+    - name: INGRESS_LOG_LEVEL
+      value: ${LOG_LEVEL}
+    - name: INGRESS_VALIDTOPICS
+      value: ${INGRESS_VALIDTOPICS} 
+    - name: INGRESS_INVENTORYURL
+      value: ${INGRESS_INVENTORYURL} 
+    - name: CLOWDER_ENABLED
+      value: "true"
+    resources:
+      limits:
+      cpu: ${CPU_LIMIT}
+      memory: ${MEMORY_LIMIT} 
+      requests:
+      cpu: 200m
+      memory: 256Mi
+    web: true
+    objectStore:
+    - ingress-uploads-perma
+    kafkaTopics:
```

Comment:
similar question as I have above. Just want to know what the purpose of this is

Reply:
That's the name of the stage bucket. Next commit will use the `INGRESS_STAGEBUCKET` and `INGRESS_REJECTBUCKET` for object store names.